statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -57,165 +69,155 @@ def ma_rep(coefs, maxn=10):
 
     .. math:: y_t = \mu + \sum_{i=0}^\infty \Phi_i u_{t-i}
 
-    e.g. can recursively compute the \Phi_i matrices with \Phi_0 = I_k
-
-    Returns
-    -------
-    phis : ndarray (maxn + 1 x k x k)
+    The MA coefficients are :math:`\Phi_i` where :math:`\Phi_0 = I_k`.
     """
     p, k, k = coefs.shape
-    phis = np.zeros((maxn+1, k, k))
+    phis = np.zeros((maxn + 1, k, k))
     phis[0] = np.eye(k)
 
     # recursively compute Phi matrices
     for i in range(1, maxn + 1):
-        for j in range(1, i+1):
+        for j in range(1, i + 1):
             if j > p:
                 break
 
-            phis[i] += np.dot(phis[i-j], coefs[j-1])
+            phis[i] += np.dot(phis[i - j], coefs[j - 1])
 
     return phis
 
 
-def is_stable(coefs, verbose=False):
+@deprecate_kwarg('verbose', 'eigenvalues')
+def is_stable(coefs, eigenvalues=False):
```

Comment:
Several of the functions in this module are redundant with methods in the classes.  `is_stable` is one that ideally I'd like to see in a `VARParams`-like class, which would be inherited by `VARProcess`.  `VARParams` would basically just have `coefs` attribute.

Reply:
The class heirarchy in VAR/VARProcess doesn't make sense.  VARPRocess needs to know about exog stuff which isn't actually in VARProcess.  It could use a big refactor but this is more than I want to try in this PR.  Ditto some of the functions (not in classes) that have `self` as the first argument so they can be used somewhere else. These should be rewritten as propper functions with lists of arguments rather than relying on a particular structure to self.

Reply:
I'm not sure exog belong in VARProcess, or at least being able to keep it out as option.
VARProcess should be the dynamic properties of the underlying VAR process, and not so much about exogenous changes in mean.